requirements/requirements-ci.txt

```diff
@@ -28,6 +28,10 @@ apipkg==1.5
     # via
     #   -r requirements-dev.txt
     #   execnet
+appnope==0.1.2
+    # via
+    #   -r requirements-dev.txt
+    #   ipython
```

Comment:
`appnope` causes problems on non-mac systems. 
It's an unfortunate issue with pip-compile on cross plattform projects.

I usually just manually delete it from the generated output.